scenarios/ocd/CreateAKSDeployment/README.md

```diff
@@ -116,124 +190,15 @@ kubectl apply -f azure-vote-start.yml
-```
-
-2. Create a Virtual Network(Vnet) for Application Gateway by running the following:
-```bash
-az network vnet create --name $VNET_NAME --resource-group $RESOURCE_GROUP_NAME --address-prefix 11.0.0.0/8 --subnet-name $SUBNET_NAME --subnet-prefix 11.1.0.0/16 
-```
-
-3. Create Application Gateway by running the following:
-
-> [!NOTE] 
-> This will take around 5 minutes 
-```bash
-az network application-gateway create --name $APPLICATION_GATEWAY_NAME --location $RESOURCE_LOCATION --resource-group $RESOURCE_GROUP_NAME --sku Standard_v2 --public-ip-address $PUBLIC_IP_NAME --vnet-name $VNET_NAME --subnet $SUBNET_NAME --priority 100
-```
-
-## Enable the AGIC add-on in existing AKS cluster 
-
-1. Store Application Gateway ID by running the following:
-```bash
-APPLICATION_GATEWAY_ID=$(az network application-gateway show --name $APPLICATION_GATEWAY_NAME --resource-group $RESOURCE_GROUP_NAME --output tsv --query "id") 
-```
-
-2. Enable Application Gateway Ingress Addon by running the following:
-
-> [!NOTE]
-> This will take a few minutes
-```bash
-az aks enable-addons --name $AKS_CLUSTER_NAME --resource-group $RESOURCE_GROUP_NAME --addon ingress-appgw --appgw-id $APPLICATION_GATEWAY_ID
-```
-
-3. Store the node resource as an environment variable group by running the following:
-```bash
-NODE_RESOURCE_GROUP=$(az aks show --name myAKSCluster --resource-group $RESOURCE_GROUP_NAME --output tsv --query "nodeResourceGroup")
-```
-4. Store the Vnet name as an environment variable by running the following:
-```bash
-AKS_VNET_NAME=$(az network vnet list --resource-group $NODE_RESOURCE_GROUP --output tsv --query "[0].name")
-```
-
-5. Store the Vnet ID as an environment variable by running the following:
-```bash
-AKS_VNET_ID=$(az network vnet show --name $AKS_VNET_NAME --resource-group $NODE_RESOURCE_GROUP --output tsv --query "id")
-```
-## Peer the two virtual networks together 
-Since we deployed the AKS cluster in its own virtual network and the Application Gateway in another virtual network, you'll need to peer the two virtual networks together in order for traffic to flow from the Application Gateway to the pods in the cluster. Peering the two virtual networks requires running the Azure CLI command two separate times, to ensure that the connection is bi-directional. The first command will create a peering connection from the Application Gateway virtual network to the AKS virtual network; the second command will create a peering connection in the other direction.
-
-1. Create the peering from Application Gateway to AKS by runnig the following:
-```bash
-az network vnet peering create --name $APPGW_TO_AKS_PEERING_NAME --resource-group $RESOURCE_GROUP_NAME --vnet-name $VNET_NAME --remote-vnet $AKS_VNET_ID --allow-vnet-access 
-```
 
-2. Store Id of Application Gateway Vnet As enviornment variable by running the following:
+>[!Note] 
+>It often takes 2-3 minutes for the PODs to be created and the site to be reachable via http
```

Comment:
Should the timeout on the curl request be modified so that the request doesn't fail before the site to become reachable?